src/temp_compare_matches_owl_and_nomenclature.py

```diff
@@ -0,0 +1,30 @@
+"""Temporary analysis file to compare exact matches to ICD11MMS between Orphanet OWL release and nomenclature files."""
```

Comment:
## Complete review for: [Temp: ORDO mappings analysis #11](https://github.com/monarch-initiative/icd11/pull/11#issue-2209486198)
- [ ] @matentzn and/or @twhetzel review

I've closed that PR because it does not need to be merged. It serves as simply a way to communicate an analysis. As of 2024/03/30, there are still 5 open discussion threads there, so I'd like to make sure those are at least looked at.